chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -352,23 +339,60 @@
     end
   end
 else
-  cookbook_file "/etc/init.d/crowbar" do
-    owner "root"
-    group "root"
-    mode "0755"
-    action :create
-    source "crowbar.suse"
-  end
+  if node["platform_version"].to_f < 12.0
+    cookbook_file "/etc/init.d/crowbar" do
+      owner "root"
+      group "root"
+      mode "0755"
+      action :create
+      source "crowbar.suse"
+    end
 
-  link "/usr/sbin/rccrowbar" do
-    action :create
-    to "/etc/init.d/crowbar"
-    not_if "test -L /usr/sbin/rccrowbar"
+    link "/usr/sbin/rccrowbar" do
+      action :create
+      to "/etc/init.d/crowbar"
+    end
+
+    # Make sure that any dependency change is taken into account
+    bash "insserv crowbar service" do
```

Comment:
Yes when the service is not enabled, but if it's already enabled and the init file changes, then we need to call insserv to update the dependency chain.